opendevin/sandbox/process.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+
+
+class Process(ABC):
+    @property
+    @abstractmethod
+    def pid(self) -> int:
+        pass
+
+    @property
+    @abstractmethod
+    def command(self) -> str:
+        pass
+
+    # @property
+    # @abstractmethod
+    # def exit_code(self) -> int | None: # The reason it can be none is because the process can still be running
+    #    pass
+
+    # @abstractmethod
+    # def kill(self):
+    #     pass
```

Comment:
```suggestion
```

Delete commented code, unless it's commented with the intention to uncomment it eventually, in which case we can add a TODO.

Reply:
Sorry, you're right, it shouldn't be there.